### ISSUE TITLE
ErrorException: Array to string conversion

### ISSUE THREAD

**Issue by iPurpl3x** [askvortsov1/flarum-pwa#31]
Flarum: v1.0.4
flarum-pwa: v3.0.5
PHP: 8.0

Found this in my logs (many times, many different days):

```
[2021-10-04 09:37:47] flarum.ERROR: ErrorException: Array to string conversion in /opt/flarum/vendor/flarum/core/src/Http/RouteCollection.php:156
Stack trace:
#0 [internal function]: FoF\Sentry\SentryServiceProvider->handleError()
#1 /opt/flarum/vendor/flarum/core/src/Http/RouteCollection.php(156): implode()
#2 /opt/flarum/vendor/flarum/core/src/Http/RouteCollectionUrlGenerator.php(46): Flarum\Http\RouteCollection->getPath()
#3 /opt/flarum/vendor/askvortsov/flarum-pwa/src/PushNotificationDriver.php(170): Flarum\Http\RouteCollectionUrlGenerator->route()
#4 /opt/flarum/vendor/askvortsov/flarum-pwa/src/PushNotificationDriver.php(91): Askvortsov\FlarumPWA\PushNotificationDriver->getPayload()
#5 /opt/flarum/vendor/flarum/core/src/Notification/NotificationSyncer.php(110): Askvortsov\FlarumPWA\PushNotificationDriver->send()
#6 /opt/flarum/vendor/fof/subscribed/src/Jobs/SendNotificationWhenUserIsCreated.php(53): Flarum\Notification\NotificationSyncer->sync()
#7 /opt/flarum/vendor/illuminate/container/BoundMethod.php(36): FoF\Subscribed\Jobs\SendNotificationWhenUserIsCreated->handle()
#8 /opt/flarum/vendor/illuminate/container/Util.php(40): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#9 /opt/flarum/vendor/illuminate/container/BoundMethod.php(93): Illuminate\Container\Util::unwrapIfClosure()
#10 /opt/flarum/vendor/illuminate/container/BoundMethod.php(37): Illuminate\Container\BoundMethod::callBoundMethod()
#11 /opt/flarum/vendor/illuminate/container/Container.php(651): Illuminate\Container\BoundMethod::call()
#12 /opt/flarum/vendor/illuminate/bus/Dispatcher.php(128): Illuminate\Container\Container->call()
#13 /opt/flarum/vendor/illuminate/pipeline/Pipeline.php(128): Illuminate\Bus\Dispatcher->Illuminate\Bus\{closure}()
#14 /opt/flarum/vendor/illuminate/pipeline/Pipeline.php(103): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#15 /opt/flarum/vendor/illuminate/bus/Dispatcher.php(132): Illuminate\Pipeline\Pipeline->then()
#16 /opt/flarum/vendor/illuminate/queue/CallQueuedHandler.php(120): Illuminate\Bus\Dispatcher->dispatchNow()
#17 /opt/flarum/vendor/illuminate/pipeline/Pipeline.php(128): Illuminate\Queue\CallQueuedHandler->Illuminate\Queue\{closure}()
#18 /opt/flarum/vendor/illuminate/pipeline/Pipeline.php(103): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#19 /opt/flarum/vendor/illuminate/queue/CallQueuedHandler.php(122): Illuminate\Pipeline\Pipeline->then()
#20 /opt/flarum/vendor/illuminate/queue/CallQueuedHandler.php(70): Illuminate\Queue\CallQueuedHandler->dispatchThroughMiddleware()
#21 /opt/flarum/vendor/illuminate/queue/Jobs/Job.php(98): Illuminate\Queue\CallQueuedHandler->call()
#22 /opt/flarum/vendor/illuminate/queue/Worker.php(428): Illuminate\Queue\Jobs\Job->fire()
#23 /opt/flarum/vendor/illuminate/queue/Worker.php(378): Illuminate\Queue\Worker->process()
#24 /opt/flarum/vendor/illuminate/queue/Worker.php(172): Illuminate\Queue\Worker->runJob()
#25 /opt/flarum/vendor/illuminate/queue/Console/WorkCommand.php(117): Illuminate\Queue\Worker->daemon()
#26 /opt/flarum/vendor/illuminate/queue/Console/WorkCommand.php(101): Illuminate\Queue\Console\WorkCommand->runWorker()
#27 /opt/flarum/vendor/illuminate/container/BoundMethod.php(36): Illuminate\Queue\Console\WorkCommand->handle()
#28 /opt/flarum/vendor/illuminate/container/Util.php(40): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#29 /opt/flarum/vendor/illuminate/container/BoundMethod.php(93): Illuminate\Container\Util::unwrapIfClosure()
#30 /opt/flarum/vendor/illuminate/container/BoundMethod.php(37): Illuminate\Container\BoundMethod::callBoundMethod()
#31 /opt/flarum/vendor/illuminate/container/Container.php(651): Illuminate\Container\BoundMethod::call()
#32 /opt/flarum/vendor/illuminate/console/Command.php(136): Illuminate\Container\Container->call()
#33 /opt/flarum/vendor/symfony/console/Command/Command.php(299): Illuminate\Console\Command->execute()
#34 /opt/flarum/vendor/illuminate/console/Command.php(121): Symfony\Component\Console\Command\Command->run()
#35 /opt/flarum/vendor/symfony/console/Application.php(996): Illuminate\Console\Command->run()
#36 /opt/flarum/vendor/symfony/console/Application.php(295): Symfony\Component\Console\Application->doRunCommand()
#37 /opt/flarum/vendor/symfony/console/Application.php(167): Symfony\Component\Console\Application->doRun()
#38 /opt/flarum/vendor/flarum/core/src/Console/Server.php(42): Symfony\Component\Console\Application->run()
#39 /opt/flarum/flarum(24): Flarum\Console\Server->listen()
```

**Comment by iPurpl3x** [comment]
@askvortsov1 could you please look at this ? it is filling up the logs since a while...

**Comment by askvortsov1** [comment]
Should be fixed in v3.1.0

**Comment by iPurpl3x** [comment]
Thanks a lot 🙏🏻 !
